### ISSUE TITLE
Gestures not working properly on map open

### ISSUE THREAD

**Issue by JenishKananiLime** [mapbox/mapbox-maps-ios#1976]
## Environment

- Xcode version:
- iOS version: 16.0+
- Devices affected:
- Maps SDK Version: 10.14.0-rc.1

## Observed behavior and steps to reproduce
Impossible to zoom out with one hand, when attempting to zoom out with one hand swipe gesture is triggered and maps pans arounds instead of zooming out. 

## Expected behavior
zoom out≈cc

## Additional links and references

https://drive.google.com/file/d/1OjoOwSkmaDHgiNNpFyzIWJaK-bWO8C2D/view?usp=sharing

**Comment by JenishKananiLime** [comment]
we are on 10.11.0 before and everything works fine it seems like after making change to zoom in and swipe at same time pr causing this issue, we can't upgrade to latest and greatest until this is fixed.

**Comment by baleboy** [comment]
Are you able to reproduce this issue with the [basic mapview example](https://github.com/mapbox/mapbox-maps-ios/blob/main/Apps/Examples/Examples/All%20Examples/BasicMapExample.swift)? Looking at the video it looks like some gestures are dropped but that seems to be a different issue.

**Comment by JenishKananiLime** [comment]
@baleboy not sure but we didn't change anything in code just simply upgrading leads to the issues in the video

**Comment by baleboy** [comment]
Here is what I can tell from the video, please correct me if I'm interpreting it wrong. In the beginning the map doesn't respond to swiping gestures with one finger. After a few attempts it does, then stops responding again and finally doesn't respond to zooming gestures.

We are not able to reproduce this behaviour in our test app and there is no obvious reason why this would happen. If by "swipe at same time PR" you are referring to [this commit](https://github.com/mapbox/mapbox-maps-ios/commit/e118546ed2e49cb60ee0195890afebdf5d42af7d), it doesn't introduce the behaviour you are referring to.

So in conclusion either I misunderstood what the problem is, or it is because of something between the application and the map view in which case we would need a project that reproduces the issue in order to continue investigating.

**Comment by JenishKananiLime** [comment]
Im sorry but why would this happen with only a map verision bump, we dont change anything in the code just bumping a version to latest causes this issue

**Comment by JenishKananiLime** [comment]
It a mapbox sdk bug with something changed between two versions thats causing this
